week4/week4-082.Lottery/src/LotteryNumbers.java

```diff
@@ -0,0 +1,35 @@
+import java.util.ArrayList;
+import java.util.Random;
+
+public class LotteryNumbers {
+    private ArrayList<Integer> numbers;
+    private int number;
+    private Random random;
+
+    public LotteryNumbers() {
+        this.numbers = new ArrayList<Integer>();
```

Comment:
Znam da sam cjepidlaka ali:

` this.numbers = new ArrayList<>();`

Prolazi bez problema, to se zove "type inference" tj. kompajler može da zaključi da je integer na osnovu tipa koji si definisao tako da je to suvišno pisati, čini kod malo čistijim - čist kod je dobar kod. :)

Isto, vidjećeš poslije, zbog polimorfizma nećeš koristiti ArrayList kao atribut već samo List interface da izbjegneš "tight coupling".

(eto ti 2 nova pojma, oba su onako, prilično napredna, ali, eto, proguglaj vidi...malo po malo... :)